### PR TITLE
Stabilize feature `macro_at_most_once_rep`

### PR DESCRIPTION
**Pull request by mark-i-m**
a.k.a. `?` Kleene operator :tada: 

cc #48075 

r? @Centril

### CODE REVIEW

File: src/libsyntax/ext/tt/quoted.rs [hunk 1]
```diff
@@ -566,32 +566,17 @@ fn parse_sep_and_kleene_op_2018<I>(
...
-    attrs: &[ast::Attribute],
+    _features: &Features,
+    _attrs: &[ast::Attribute],
 ) -> (Option<token::Token>, KleeneOp)
 where
     I: Iterator<Item = tokenstream::TokenTree>,
 {
     // We basically look at two token trees here, denoted as #1 and #2 below
     let span = match parse_kleene_op(input, span) {
         // #1 is a `?` (needs feature gate)
```
Comment (Centril) [inline src/libsyntax/ext/tt/quoted.rs]:
```suggestion
         // #1 is a `?`
```

**Review by Centril — CHANGES_REQUESTED** [review]
Pending tests, everything except a comment looks good. :)

**Review by Centril — APPROVED** [review]

**Comment by mark-i-m** [comment]
Woah, you're fast.

Thanks!

**Comment by mark-i-m** [comment]
Travis is passing, so i guess we just wait for FCP in 4 days?

**Comment by Centril** [comment]
@mark-i-m I don't have r+ myself so maybe @petrochenkov or @eddyb could r+; we could also wait those 4 days but I wouldn't mind just merging now. ^.^

**Comment by mark-i-m** [comment]
Oh, it's interesting that rust-highfive bot didn't complain.

r? @petrochenkov

**Comment by pietroalbini** [comment]
@bors retry

Yielding priority to https://github.com/rust-lang/rust/pull/56313, which fixes a bad nightly regression.

**Comment by mark-i-m** [comment]
@Centril Should this have had the `relnotes` tag?

**Comment by Centril** [comment]
@mark-i-m oh weird; I thought I added the `relnotes` tag... :P

**Pull request merged**